ci-operator/config/openshift-priv/cluster-network-operator/openshift-priv-cluster-network-operator-master.yaml

```diff
@@ -133,15 +133,16 @@ tests:
     cluster_profile: vsphere
     workflow: openshift-e2e-vsphere-ovn
 - as: e2e-vsphere-ovn-dualstack
+  optional: true
+  steps:
+    cluster_profile: vsphere
+    workflow: openshift-e2e-vsphere-ovn-dualstack
+- as: e2e-vsphere-ovn-dualstack-primaryv6
   optional: true
   steps:
     cluster_profile: vsphere
     env:
-      TEST_SKIPS: Services Secondary IP Family \[LinuxOnly\] should function for pod-Service\|
```

Comment:
Can we please just add new jobs in this PR and not modify the existing ones? There is a separate work being done to stop skipping some tests, but this PR says explicitly that it "add DualStackIPv6Primary for vSphere" so we should not modify the scope of conformance tests at the same time

Of course the jobs that you are adding here do not need any `TEST_SKIPS`

Reply:
yep, bad copy-paste